wremnants/CardTool.py

```diff
@@ -923,7 +923,7 @@ def writeHist(self, h, proc, syst, setZeroStatUnc=False, decorrByBin={}, hnomi=N
             return
         if self.project:
             axes = self.project[:]
-            if "charge" in h.axes.name and not self.xnorm:
+            if "charge" in h.axes.name and not self.xnorm and "charge" not in axes:
```

Comment:
I think there is a self.charge_ax (or there should be). We should avoid hardcoding it